RELEASE/scripts/autoscend/autoscend_header.ash

```diff
@@ -271,6 +271,18 @@ boolean getDiscoStyle();
 boolean volcano_lavaDogs();
 boolean volcano_bunkerJob();
 
+//Defined in autoscend/iotms/auto_eudora.ash
+boolean eudora_available();
+boolean[item] eudora_initializeSettings();
+item eudora_current();
+int[item] eudora_xiblaxian();
+
+//Defined in autoscend/iotms/auto_floristfriar.ash
+void florist_initializeSettings();
+boolean didWePlantHere(location loc);
+void trickMafiaAboutFlorist();
+void oldPeoplePlantStuff();
```

Comment:
Is there some reason this stuff isn't included in mr2013.ash? If not, maybe we should move them there and get rid of the extra file.

Reply:
change made